internal/controller/controllers/agent_controller.go

```diff
@@ -440,13 +467,28 @@ func (r *AgentReconciler) updateStatus(ctx context.Context, log logrus.FieldLogg
 			agent.Status.ValidationsInfo = newValidationsInfo
 		}
 
-		if h.Progress != nil && h.Progress.CurrentStage != "" {
-			if isNoneDay2Rebooting, err = r.isDay2NonePlatformHostRebooting(ctx, agent, h); err != nil {
-				log.WithError(err).Errorf("Failed to find if agent %s/%s belongs to none platform cluster and is rebooting", agent.Namespace, agent.Name)
-				return ctrl.Result{RequeueAfter: defaultRequeueAfterOnError}, nil
+		// Determine whether or not CSRs should be automatically signed.
+		if cluster != nil {
```

Comment:
We have already many places checking for SNO:
https://github.com/openshift/assisted-service/blob/6382aa70347ac0e31c1dfbd6ed957be88d357acc/internal/controller/controllers/clusterdeployments_controller.go#L1074

And

https://github.com/openshift/assisted-service/blob/6382aa70347ac0e31c1dfbd6ed957be88d357acc/internal/common/common.go#L113

Why another one ?